src/frontend/src/components/RecordSearch/types.ts

```diff
@@ -0,0 +1,7 @@
+export interface AliasType {
```

Comment:
Why not just Alias?

Reply:
We call every Typescript class a `SomethingType` which seems not great but it's because there are React Components we would want to also call by the same, unless we call those `SomethingComponent` or such

Reply:
I'd be excited about a nicer approach